crates/models/src/materializations.rs

```diff
@@ -16,7 +16,8 @@ use std::collections::BTreeMap;
 pub struct MaterializationDef {
     /// # Automatically materialize new bindings from a named capture
     #[serde(skip_serializing_if = "Option::is_none")]
-    pub source_capture: Option<SourceCapture>,
+    #[serde(alias = "sourceCapture")]
+    pub sources: Option<Sources>,
```

Comment:
this is an annoying change request, but I think it should be `source` instead of `sources`. This seems more consistent with how we pluralize things elsewhere in models, including with the `source` field within a binding.

Then we get:

```
source:
  capture: acmeCo/capture
  targetNaming: noSchema
```

```
source:
  targetNaming: withSchema
```

I think `sources: { capture: acmeCo/capture }` is confusingly inconsistent with `source: acmeCo/collection` under a binding -- I can't explain to a user why they're different -- and `source: {targetNaming: withSchema}` also seems fine.

Reply:
Trying to frame the counter argument / explanation:

`sources` is configuration with respect to _all_ sources of the materialization. `sources: { capture: acmeCo/capture }` is appropriate because that knob affects all materialization bindings and their (singular) `source`.

Reply:
> would you still feel the same if it were

I think `source` is more appropriate than `sources` in that context, though `captures` makes sense pluralized.

> AFAIK we'd still like to eventually allow specifying multiple source captures.

I think this is still a matter for debate. I currently lean the other way.